client-js/app/index.html

```diff
@@ -19,7 +19,7 @@
   <!-- Add to homescreen for Chrome on Android -->
   <meta name="mobile-web-app-capable" content="yes">
   <meta name="application-name" content="LabRAD">
-  <link rel="icon" sizes="192x192" href="/images/touch/chrome-touch-icon-192x192.png">
+  <link rel="icon" href="/images/labrad.png">
```

Comment:
Maybe something like the following (stolen from the Grapher):
![labrad3](https://cloud.githubusercontent.com/assets/8183919/13544095/f6f7d3de-e226-11e5-8e77-c39c7f12dd1a.png)